src/test/java/com/scottlogic/utils/UserPostFilterData.java

```diff
@@ -27,6 +27,7 @@
     private static UserPost userPost5 = new UserPost("This funky-name",
             OffsetDateTime.of(2001, 1, 1, 1, 1, 1, 0, ZoneOffset.UTC),
             "funky funk funk", 0);
+
     private static UserPost userPost6 = new UserPost("steve madden",
```

Comment:
Maybe this is a comment to be added to any refactor work you do:

Can these objects be final? If so, convention is to use all uppercase for naming: USER_POST_6

Reply:
I did not know about this keyword - implemented